From 252cd9b20d7857091c0529955998b86d7a90e6c5 Mon Sep 17 00:00:00 2001
From: Hubert Figuiere <hub@figuiere.net>
Date: Thu, 22 Jul 2010 07:04:54 +0000
Subject: Install the GdkPixbuf in the right place. (Closes #29208) (Vincent Untz)

---
diff --git a/configure.in b/configure.in
index 1d44b9c..c16a20c 100644
--- a/configure.in
+++ b/configure.in
@@ -137,8 +137,7 @@ if test x$enable_gnome = xyes ; then
    if test x$HAVE_LIBGLIB = xyes ; then
       PKG_CHECK_MODULES(LIBGDKPIXBUF, gdk-pixbuf-2.0 >= $LIBGDKPIXBUF_REQUIRED,
                      [AC_DEFINE(HAVE_LIBGDKPIXBUF, 1, [Define to 1 to enable libgdkpixbuf support])
-		      GTK_VERSION=`$PKG_CONFIG --variable=gtk_binary_version gtk+-2.0`
-		      GDK_PIXBUF_DIR="gtk-2.0/$GTK_VERSION/loaders"
+		      GDK_PIXBUF_DIR=`$PKG_CONFIG --variable=gdk_pixbuf_moduledir gdk-pixbuf-2.0`
 		      AC_SUBST(GDK_PIXBUF_DIR)
 		      HAVE_LIBGDKPIXBUF=yes],
                      [HAVE_LIBGDKPIXBUF=no])
diff --git a/gnome/Makefile.am b/gnome/Makefile.am
index ecfe27d..2c0bca5 100644
--- a/gnome/Makefile.am
+++ b/gnome/Makefile.am
@@ -16,7 +16,7 @@ pkgconfig_DATA = $(PKG_CONFIG_FILES)
 
 lib_LTLIBRARIES = $(GNOME_TARGETS)
 
-loaderdir = $(libdir)/$(GDK_PIXBUF_DIR)
+loaderdir = $(GDK_PIXBUF_DIR)
 loader_LTLIBRARIES = $(GDK_PIXBUF_PLUGIN)
 
 libopenrawgnome_la_LDFLAGS = \
--
cgit v0.8.3-6-g21f6
From d4061df829ede099e9e5093e9d33f30c4a9f7783 Mon Sep 17 00:00:00 2001
From: Hubert Figuiere <hub@figuiere.net>
Date: Sun, 05 Sep 2010 02:25:39 +0000
Subject: Fix detection of the module dir for GdkPixbuf. (Closes: #30013)

This is a regression caused by a previous commit.
---
diff --git a/configure.in b/configure.in
index c16a20c..a9a1a80 100644
--- a/configure.in
+++ b/configure.in
@@ -138,6 +138,12 @@ if test x$enable_gnome = xyes ; then
       PKG_CHECK_MODULES(LIBGDKPIXBUF, gdk-pixbuf-2.0 >= $LIBGDKPIXBUF_REQUIRED,
                      [AC_DEFINE(HAVE_LIBGDKPIXBUF, 1, [Define to 1 to enable libgdkpixbuf support])
 		      GDK_PIXBUF_DIR=`$PKG_CONFIG --variable=gdk_pixbuf_moduledir gdk-pixbuf-2.0`
+			dnl This is because the variable is only defined on recent gdk-pixbuf. 
+			dnl Revert to the old method otherwise. (https://bugs.freedesktop.org/show_bug.cgi?id=30013)
+			if test -z "$GDK_PIXBUF_DIR" ; then
+				GTK_VERSION=`$PKG_CONFIG --variable=gtk_binary_version gtk+-2.0`
+				GDK_PIXBUF_DIR="$libdir/gtk-2.0/$GTK_VERSION/loaders"
+			fi
 		      AC_SUBST(GDK_PIXBUF_DIR)
 		      HAVE_LIBGDKPIXBUF=yes],
                      [HAVE_LIBGDKPIXBUF=no])
--
cgit v0.8.3-6-g21f6
