--- libopenraw-0.3.7/lib/Makefile.am.orig	2023-07-30 00:00:00.000000000 +0200
+++ libopenraw-0.3.7/lib/Makefile.am	2023-07-30 00:00:01.000000000 +0200
@@ -121,9 +121,8 @@
 # The use of --all-static is necesary for the RUST_LIB
 libopenraw_la_LDFLAGS = \
 	-version-info @LIBOPENRAW_VERSION_INFO@ \
-	--all-static \
-	-lpthread -ldl
-libopenraw_la_LIBADD = -ljpeg $(RUST_LIB) libopenraw_internals.la
+	--all-static
+libopenraw_la_LIBADD = -ljpeg $(RUST_LIB) libopenraw_internals.la -lpthread -ldl
 libopenraw_la_SOURCES = \
 	capi/capi.h \
 	capi/capi.cpp \
