--- libopenraw-0.3.7/lib/Makefile.am.orig	2023-07-30 00:00:00.000000000 +0200
+++ libopenraw-0.3.7/lib/Makefile.am	2023-07-30 00:00:01.000000000 +0200
@@ -104,7 +104,7 @@
 endif

 CARGO_TARGET_DIR=@abs_top_builddir@/lib/mp4/target
-RUST_PATH=$(CARGO_TARGET_DIR)/@CARGO_TARGET_SUBDIR@
+RUST_PATH=$(CARGO_TARGET_DIR)/$(CARGO_TARGET_SUBDIR)
 RUST_LIB=$(RUST_PATH)/libmp4.a

 mp4/mp4parse.h: $(RUST_LIB)
